src/structured-data/schema.js

```diff
@@ -0,0 +1,64 @@
+export const schemaContext = { "@context": "http://schema.org" }
+
+export function schemaType(type) {
+  return { "@type": type };
+}
+
+export function schemaPerson(name) {
+  return Object.assign({},
+    schemaType("Person"),
+    {
+      "givenName": name,
+      "name": name
+    }
+  )
+}
+
+export function schemaBlogPosting(card, author, headline, image, structuredData, story) {
+  return Object.assign({},
+    schemaType("BlogPosting"),
+    schemaMainEntityOfPage(structuredData.website.url + story.slug),
```

Comment:
interpolation is much more performant than concatination, `${structuredData.website.url}${story.slug}` maybe

Reply:
let's limit object access to second level, to ensure it wont break if the second level value is undefined. 

The workaround is use to destructing or a variable to do the same.


instead of `structuredData.website.url`, we can proceed to use

```
const {website : {url = ''} = {}} = structuredData
```